src/typescript.js

```diff
@@ -28,7 +28,7 @@ export default [
     languageOptions: {
       parserOptions: {
         projectService: true,
-        tsconfigRootDir: import.meta.dirname
+        tsconfigRootDir: process.cwd()
```

Comment:
In https://github.com/typescript-eslint/typescript-eslint/issues/6883#issuecomment-1504826998, the maintainer of `typescript-eslint` states that

> [...] you should never set tsconfigRootDir from within a shared config [...]

He also [mentions](https://github.com/typescript-eslint/typescript-eslint/issues/6883#issuecomment-1504627866) that

> The default value for tsconfigRootDir is process.cwd() - so explicitly setting it to process.cwd() is the same as not setting it at all.

See also [this](https://github.com/typescript-eslint/typescript-eslint/blob/f83a5910569a9566a452ecac8f8843370a413f3c/packages/typescript-estree/src/parseSettings/createParseSettings.ts#L49) line in the source code.

I haven't looked deeply into this, but are you sure that this is the correct fix?

Reply:
Thanks for the links! Digging a bit deeper into this, I found that running `eslint src` works, while `eslint .` fails for some reason due to `parserOptions` not being detected in Horusnome.

I've moved to `eslint src` there to get this working now.